Boost 1.89 removed boost_system

See also:
https://github.com/boostorg/system/commit/7a495bb46d7ccd808e4be2a6589260839b0fd3a3
https://github.com/scottdraves/electricsheep/issues/131
https://github.com/stefantalpalaru/gentoo-overlay/issues/182

Pascal Jaeger <pascal.jaeger@leimstift.de> (2026-02-06)

diff '--color=auto' -ur client_generic.orig/Client/Makefile.am client_generic/Client/Makefile.am
--- client_generic/Client/Makefile.am	2024-02-13 15:55:46.000000000 +0100
+++ client_generic/Client/Makefile.am	2026-02-01 14:14:47.781773546 +0100
@@ -75,7 +75,7 @@
 
 
 
-electricsheep_LDADD = -lboost_system -lboost_thread -lboost_filesystem -lglut \
+electricsheep_LDADD = -lboost_thread -lboost_filesystem -lglut \
 	$(AVCODEC_LIBS) $(AVFORMAT_LIBS) $(SWSCALE_LIBS) $(AVUTIL_LIBS) $(LUA_LIBS) $(GLU_LIBS) $(GLEE_LIBS) $(BOOST_LDADD) \
 	$(CURL_LIBS) $(PNG_LIBS) $(XRENDER_LIBS) $(LIBGTOP_LIBS) $(XRENDER_LIBS)
 
diff '--color=auto' -ur client_generic.orig/configure.ac client_generic/configure.ac
--- client_generic/configure.ac	2026-02-01 14:13:46.216373226 +0100
+++ client_generic/configure.ac	2026-02-01 14:14:31.813684871 +0100
@@ -278,15 +278,11 @@
 	[AC_MSG_ERROR([you must install libboost-system to compile electricsheep.])
 ])
 
-AC_CHECK_LIB([boost_system],[main],,
-	[AC_MSG_ERROR([you must install libboost-system to compile electricsheep.])
-])
 
 AC_LANG_POP([C++])
 
 
 
-BOOST_LDADD="-lboost_system"
 
 AC_SUBST(BOOST_LDADD)
 
